### PR TITLE
Create students data

### LINKED ISSUE #12: Allow users to add students data
Allow users to add students data

Fields: https://github.com/AY2021S1-CS2103T-F11-1/tp/issues/2

Existing fields which AB3 provides:
- Name
- NUS Email
- Address
- Tags

Required fields:
- Telegram
- Matric number

### PR DESCRIPTION
**Pull request by kwannoel**
### Details
See commit log for details.
Student data added according to #2
Closes #12 

### TODOs (after this gets merged):
Currently CI is failing because:
- #13, #14, #15 Have to update definitions of `new Person(...)`
- #42 After the above issues are resolved, run linting checks and tests through `gradle`.

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -20,21 +16,33 @@
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
+                new Telegram("alexyeoh"),
```
Comment (Rishi5154) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:25]: A minor thing, but perhaps the handle has to start with @ right?

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 2]
```diff
@@ -12,11 +14,7 @@
 
 import seedu.address.logic.commands.AddCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```
Comment (chrisgzf) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java]: HALLO POLIS?

**Review by chrisgzf — CHANGES_REQUESTED** [review]
Please try to avoid wildcard imports.

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (kwannoel) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:25]: For input I think we can leave it as that, without the prefix `@`.

I have updated the `toString` instance to reflect the handle.

**Pull request merged**